src/plugins/intel_gpu/src/graph/impls/onednn/utils.cpp

```diff
@@ -817,5 +817,50 @@ bool is_supported_pad(const layout& layout) {
     return (no_spatial_padding && no_batch_padding);
 }
 
+int onednn_post_ops_fusing_helpers::get_prelu_mask_from_layouts(const std::function<layout()>& get_output_layout,
+                                                                const std::function<layout(int32_t)>& get_input_layout,
+                                                                int32_t slope_input_idx) {
+    auto data_layout = get_output_layout();
+    auto slope_layout = get_input_layout(slope_input_idx);
+    auto input_layout = get_input_layout(0);
+    auto data_shape = data_layout.get_shape();
+    auto slope_shape = slope_layout.get_shape();
+    auto input_shape = input_layout.get_shape();
+    int ndims = slope_shape.size() == 1 ? static_cast<int>(input_shape.size()) : static_cast<int>(slope_shape.size());
+
+    bool is_scalar = true;
+    bool is_per_tensor = true;
+    for (size_t i = 0; i < slope_shape.size(); i++) {
+        if (slope_shape[i] != 1)
+            is_scalar = false;
+        if (slope_shape[i] != data_shape[i])
+            is_per_tensor = false;
+    }
+
+    if (is_scalar)
+        return get_default_mask(post_op_dnnl_policy_type::COMMON, ndims);
```

Comment:
I think there should be no `scalar` case for prelu. If it is a scalar, it is better handled by leaky relu: [dnnl_eltwise_relu](https://uxlfoundation.github.io/oneDNN/enum_dnnl_alg_kind_t.html#doxid-group-dnnl-api-primitives-common-1gga96946c805f6c4922c38c37049ab95d23a5e37643fec6531331e2e38df68d4c65a)